### DISCUSSION TITLE
xeol in mixed OS environment

### DISCUSSION THREAD

**Discussion by goeldicom** [xeol-io/xeol#213] [General]
Although I am a 100% Linux guy, I want to use xeol for the windows servers of the Microsoft Fanboys in my University.

Is there a way for this?

I thought of steps like these, but there may be better ways:

- using wsl to mount the windows partition into wsl and run xeol from inside wsl
- creating a SBOM on the windows partition and externally xeol-scanning the SBOM

**Comment by noqcks** [comment]
excuse my ignorance, because I haven't used windows in quite some time, and haven't tested xeol on a windows machine, but we do have a windows release on the release page. https://github.com/xeol-io/xeol/releases/tag/v0.9.6

you should be able to run this on windows without using wsl? This should be supported just as its [supported with syft](https://github.com/anchore/syft/pull/548)
